lib/active_merchant/billing/gateways/paymentez.rb

```diff
@@ -219,7 +219,11 @@ def message_from(response)
         if success_from(response)
           response['transaction'] && response['transaction']['message']
         else
-          response['error'] && response['error']['type']
+          if response['error']
+            response['error'] && response['error']['type']
+          else
+            response['transaction']['message']
```

Comment:
It might be worth considering concatenating both the error type and transaction message in this case, which we do for some other gateways.

Reply:
Might be misunderstanding, but it looks like its one or the other. We get back an error or a transaction.